### PR TITLE
Feat/widget wysiwyg

### PR DESCRIPTION
**Pull request by yoslie**
## Overview
Slicing layout and create preview component widget wysiwyg

### Preview
https://github.com/jabardigitalservice/j-site-public/assets/65396086/8268baf5-6c76-4b92-8ce6-473ede77465f

### Evidence
title: Slicing layout and create preview component widget wysiwyg
project: J-Site
participants: @yoslie @doohanas @Ibwedagama

### CODE REVIEW

File: components/Widget/Wysiwyg.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="wysiwyg flex h-full w-full flex-col">
+    <ClientOnly>
+      <!-- eslint-disable vue/no-v-html -->
+      <div v-html="props.payload.content"></div>
+      <!--eslint-enable-->
+    </ClientOnly>
```
Comment (Ibwedagama) [inline components/Widget/Wysiwyg.vue:7]: ini kenapa pake ClientOnly yoss? @yoslie

File: components/Widget/Wysiwyg.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+<style>
+  .wysiwyg {
+    h2,
+    h3,
+    h4,
+    h5 {
+      @apply font-inter font-semibold;
+    }
+
```
Comment (Ibwedagama) [inline components/Widget/Wysiwyg.vue:30]: ini kenapa stylenya gk dibuat scoped yos @yoslie ? ada isu kah?

File: components/Widget/Wysiwyg.vue [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+<script setup lang="ts">
+  const props = defineProps({
+    payload: {
+      type: Object,
+      default: () => ({
+        content: '',
+      }),
+    },
+  })
```
Comment (Ibwedagama) [inline components/Widget/Wysiwyg.vue:19]: payloadnya bisa minta tolong di validasi dulu @yoslie ?

File: components/Widget/Wysiwyg.vue (hunk 2 above)
Comment (yoslie) [reply components/Widget/Wysiwyg.vue:28]: yes, the styles not override when use scoped style

File: components/Widget/Wysiwyg.vue (hunk 1 above)
Comment (yoslie) [reply components/Widget/Wysiwyg.vue]: my bad, it's unnecesarry, removed [here](https://github.com/jabardigitalservice/j-site-public/pull/30/commits/6670fe45a7ec2fcdddc69f8207cd0ac8e589a160)

File: components/Widget/Wysiwyg.vue (hunk 3 above)
Comment (yoslie) [reply components/Widget/Wysiwyg.vue:19]: sure thanks, updated [here](https://github.com/jabardigitalservice/j-site-public/pull/30/commits/a494b7bb6f65886c55519f3474c52c4505b9a438)

**Review by doohanas — APPROVED** [review]

File: components/Widget/Wysiwyg.vue (hunk 2 above)
Comment (Ibwedagama) [reply components/Widget/Wysiwyg.vue:47]: udah cobain pake .wysiwyg::v-deep @yoslie ?

**Review by Ibwedagama — APPROVED** [review]

File: components/Widget/Wysiwyg.vue (hunk 2 above)
Comment (yoslie) [reply components/Widget/Wysiwyg.vue]: ah I see, it works, thanks again man 🤟

**Pull request merged**